pkg/standalone/uninstall.go

```diff
@@ -1,24 +1,24 @@
 package standalone
 
 import (
+	"errors"
 	"fmt"
-	"strings"
 
 	"github.com/dapr/cli/pkg/rundata"
 	"github.com/dapr/cli/utils"
 )
 
 // Uninstall deletes all installed containers
 func Uninstall(uninstallAll bool, dockerNetwork string) error {
-	var failedContainers []string
+	var errs []error
 
 	_, err := utils.RunCmdAndWait(
 		"docker", "rm",
 		"--force",
 		utils.CreateContainerName(DaprPlacementContainerName, dockerNetwork))
 
 	if err != nil {
-		failedContainers = append(failedContainers, DaprPlacementContainerName)
+		errs = append(errs, fmt.Errorf("could not remove %s: %s", DaprPlacementContainerName, err))
```

Comment:
I think it's better to have "...remove %s **container** : %s" in the error message like you had before. This adds some clarity of what we are trying to remove.

Reply:
Agreed.